### PR TITLE
test: add scenarios for editing basic data of listing

### PR DESCRIPTION
**Pull request by iyerashwin14**

### CODE REVIEW

File: src/test/resources/Features/OCD-2834-SimplifiedEditingOfListingBasicData.feature [hunk 1]
```diff
@@ -29,12 +29,104 @@ Scenario: User logs in as ROLE and edits a listing
     When I am on listing details page of listing with ID "5737"
     And I click on edit certified product link
     And I set the Mandatory Disclosures URL filed to "www.example.com"
-    Then I see "Improper format" error message for Mandatory Disclosures URL filed 
+    Then I see "Improper format" error message for Mandatory Disclosures URL filed
```
Comment (andlar) [inline src/test/resources/Features/OCD-2834-SimplifiedEditingOfListingBasicData.feature]:
```suggestion
    Then I see "Improper format" error message for Mandatory Disclosures URL field
```

File: src/test/resources/Features/OCD-2834-SimplifiedEditingOfListingBasicData.feature [hunk 2]
```diff
@@ -29,12 +29,104 @@ Scenario: User logs in as ROLE and edits a listing
...
+    Given I'm logged in as "<ROLE>"
+    When I am on listing details page of listing with ID "<listing id>"
+    And I click on edit certified product link
+    And I set the product code field to "<value>"
+    Then I see error message "<Error Message>" for CHPL Product Number
+    Examples:
+      |ROLE      |listing id|value|Error Message                                                                     |
+      |ROLE_ADMIN|9268      |a    |Product Code must consist of letters, numbers and/or "_", and be 4 characters long|
+      |ROLE_ONC  |9268      |ab   |Product Code must consist of letters, numbers and/or "_", and be 4 characters long|
+      |ROLE_ACB  |9268      |abc  |Product Code must consist of letters, numbers and/or "_", and be 4 characters long|
```
Comment (andlar) [inline src/test/resources/Features/OCD-2834-SimplifiedEditingOfListingBasicData.feature]: Good examples, but should also check with more than 4 characters, and one with four characters but that has bad values, like `AB#$` or something

Should also do something similar with Version and Product codes

File: src/test/resources/Features/OCD-2834-SimplifiedEditingOfListingBasicData.feature [hunk 3]
```diff
@@ -29,12 +29,104 @@ Scenario: User logs in as ROLE and edits a listing
...
+Scenario Outline: Verify error message is displayed when one or more than two numbers entered in the ICS field for the CHPL Product Number
+    Given I'm logged in as "<ROLE>"
+    When I am on listing details page of listing with ID "<listing id>"
+    And I click on edit certified product link
+    And I set the ICS field to "<value>"
+    Then I see error message "<Error Message>" for CHPL Product Number
+    Examples:
+      |ROLE      |listing id|value|Error Message                                                                     |
+      |ROLE_ADMIN|9268      |1    |ICS Code must be a two digit single number from 00 to 99                          |
+      |ROLE_ONC  |9268      |123  |ICS Code must be a two digit single number from 00 to 99                          |
```
Comment (andlar) [inline src/test/resources/Features/OCD-2834-SimplifiedEditingOfListingBasicData.feature:106]: Should check for letters / non-numbers

**Review by andlar — COMMENTED** [review]
Looks really good. Only a couple of suggestions.

On the ones with the bad characters / length things, I added requests for some different kinds of bad data. If it had to be entire new scenarios or steps or something, I wouldn't have, but because they'll only be a few lines in the Feature file it should be easy

**Review by andlar — APPROVED** [review]

**Review by ashwinimore — APPROVED** [review]

**Pull request merged**